src/cnlpt/train_system.py

```diff
@@ -82,6 +108,108 @@ def is_hub_model(model_name: str) -> bool:
     return False
 
 
+def get_dataset_segment(
+    split_name: str,
+    dataset_ind: int,
+    dataset: ClinicalNlpDataset,
+):
+    start_ind = end_ind = 0
+    for ind in range(dataset_ind):
+        start_ind += len(dataset.datasets[ind][split_name])
+    end_ind = start_ind + len(dataset.datasets[dataset_ind][split_name])
+
+    return Dataset.from_dict(dataset.processed_dataset[split_name][start_ind:end_ind])
+
+
+def restructure_prediction(
```

Comment:
move these 2 functions to cnlp_predict?

Reply:
Moved restructure_prediction and structure_labels to cnlp_predict